bpf/bpf_wireguard.c

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Cilium */
+
+#define ETH_HLEN 0
+#define IS_BPF_WIREGUARD 1
+#define ENABLE_WIREGUARD 1
```

Comment:
Do we need the `ENABLE_WIREGUARD` ? It should be part of the `node_config.h`, no?